remove compile failed objects


Index: lmbench-3.0-a9/src/Makefile
===================================================================
--- lmbench-3.0-a9.orig/src/Makefile
+++ lmbench-3.0-a9/src/Makefile
@@ -93,14 +93,14 @@ ASMS =  $O/bw_file_rd.s $O/bw_mem.s $O/b
 	$O/rhttp.s $O/timing_o.s $O/tlb.s $O/stream.s			\
 	$O/cache.s $O/lat_dram_page.s $O/lat_pmake.s $O/lat_rand.s	\
 	$O/lat_usleep.s $O/lat_cmd.s
-EXES =	$O/bw_file_rd $O/bw_mem $O/bw_mmap_rd $O/bw_pipe $O/bw_tcp 	\
+EXES =	$O/bw_file_rd $O/bw_mem $O/bw_mmap_rd $O/bw_pipe                \
 	$O/bw_unix $O/hello						\
-	$O/lat_select $O/lat_pipe $O/lat_rpc $O/lat_syscall $O/lat_tcp	\
-	$O/lat_udp $O/lat_mmap $O/mhz $O/lat_proc $O/lat_pagefault	\
-	$O/lat_connect $O/lat_fs $O/lat_sig $O/lat_mem_rd $O/lat_ctx	\
+	$O/lat_pipe $O/lat_syscall                              	\
+	$O/lat_mmap $O/mhz $O/lat_proc $O/lat_pagefault         	\
+	$O/lat_fs $O/lat_sig $O/lat_mem_rd $O/lat_ctx           	\
 	$O/lat_sem 							\
 	$O/memsize $O/lat_unix $O/lmdd $O/timing_o $O/enough		\
-	$O/msleep $O/loop_o $O/lat_fifo $O/lmhttp $O/lat_http		\
+	$O/msleep $O/loop_o $O/lat_fifo		                        \
 	$O/lat_fcntl $O/disk $O/lat_unix_connect $O/flushdisk		\
 	$O/lat_ops $O/line $O/tlb $O/par_mem $O/par_ops 		\
 	$O/stream
